src/base/bittorrent/session.cpp

```diff
@@ -4170,10 +4175,17 @@ void Session::handleFileErrorAlert(libt::file_error_alert *p)
     // NOTE: Check this function!
     TorrentHandle *const torrent = m_torrents.value(p->handle.info_hash());
     if (torrent) {
-        QString msg = QString::fromStdString(p->message());
-        Logger::instance()->addMessage(tr("An I/O error occurred, '%1' paused. %2")
-                           .arg(torrent->name(), msg));
-        emit fullDiskError(torrent, msg);
+        const InfoHash hash = torrent->hash();
+        if (!m_recentErroredTorrents.contains(hash)) {
+            QString msg = QString::fromStdString(p->message());
```

Comment:
const QString
@Chocobo1 will insist...

Reply:
ok, but it gets a bit silly to have to update code that I simply indent one level